src/InputArea/InputArea.js

```diff
@@ -237,6 +243,11 @@ class InputArea extends React.PureComponent {
       computedStyle.getPropertyValue('line-height'),
       10,
     );
+
+    if (isNaN(fontSize)) {
+      return InputArea.MIN_ROWS;
+    }
+
     const lineHeightValue = isNaN(lineHeight)
       ? this._getDefaultLineHeight() * fontSize
       : lineHeight;
```

Comment:
From what i see here `fontSize` will be used only when the `isNaN(lineHeight)` is true, hence we shouldn't return  the `MIN_ROWS` value so early here.
I would go with a more defensive approach:
```suggestion
    let lineHeightValue;
    if (isNaN(lineHeight)) {
      if (isNaN(fontSize)) {
        return InputArea.MIN_ROWS;
      } else {
        lineHeightValue = this._getDefaultLineHeight() * fontSize;
      }
    } else {
      lineHeightValue = lineHeight;
    }
```

Reply:
Done 👍